test/helpers/assert.js

```diff
@@ -23,13 +24,24 @@ export const assertNonExistent = (t, {name, exitCode, signalName, command, messa
 	assertDurationMs(t, durationMs);
 };
 
+// Support localized error messages on non-English Windows.
+let nonExistentCommandOutput;
+const toNonExistentCommandOutput = command => {
+	if (typeof nonExistentCommandOutput !== 'string') {
+		const {stderr} = spawnSync(nonExistentCommand, {shell: true, encoding: 'utf8'});
```

Comment:
Small question: is `shell: true` necessary here? I am wondering whether the error message would show up even without that option?